### PR TITLE
[DGPFUS-3] Develop a Contact Form Page Using HTML

### PR DESCRIPTION
**Pull request by amadimeka**
html-form

!

### CODE REVIEW

File: submissions/emekaAmadi/HTML-FORMS/contactForm.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+          <br />
+          <label for="subject">Subject</label>
+          <input
+            type="text"
+            id="subject"
+            name="subject"
+            placeholder="Enter Subject"
+          />
+          <br /><br />
+          <label for="message">Message</label>
```
Comment (ToluEnny24) [inline submissions/emekaAmadi/HTML-FORMS/contactForm.html:38]: Message should have a type textarea with rows and  columns to accommodate message

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
@amadimeka please effect the changes.

File: submissions/emekaAmadi/HTML-FORMS/contactForm.html (hunk 1 above)
Comment (amadimeka) [reply submissions/emekaAmadi/HTML-FORMS/contactForm.html]: ok ma

**Comment by ToluEnny24** [comment]
@amadimeka Why did you edit PaulEzemiwe folder?. You are not to tamper with another scholar's file no matter what.

**Review by ToluEnny24 — APPROVED** [review]

**Pull request merged**